src/components/dev-popover/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useRef, useState } from 'react'
+import Popover from 'components/popover'
+
+/**
+ * Renders non-interactive children into a button element,
+ * and displays a popover when the button is activated.
+ *
+ * Intended for use with work-in-progress elements.
+ * Should not be used in production.
+ */
+function DevPopover({
+  children,
+  note,
+  title = 'Note',
+}: {
+  children: React.ReactNode
+  note: React.ReactNode
+  title?: string
+}): React.ReactElement {
+  const [showDialog, setShowDialog] = useState(false)
+  const triggerRef = useRef(null)
+  return (
+    <>
+      <button
+        style={{
+          border: 'none',
+          background: 'none',
+          fontSize: 'inherit',
+          fontFamily: 'inherit',
+          padding: 0,
+          margin: 0,
+          display: 'block',
+          cursor: 'pointer',
+          color: 'inherit',
```

Comment:
Are these styles inline so that they're hard to overwrite?

Reply:
Naw, just inline cause it's a dev component and I was too lazy to split them out 😅 I'll patch this up now!

Reply:
✅ Fixed this up!